contracts/concordium-governance-committee-election/src/lib.rs

```diff
@@ -576,11 +584,48 @@ fn post_election_result(ctx: &ReceiveContext, host: &mut Host<State>) -> Result<
     let candidates: Vec<_> = host.state.candidates.iter().collect();
     let parameter: PostResultParameter = ctx.parameter_cursor().get()?;
     ensure!(parameter.len() == candidates.len(), Error::Malformed);
-
     *host.state.election_result.get_mut() = Some(parameter);
     Ok(())
 }
 
+/// The parameter supplied to the [`reset_finalization_phase`] entrypoint.
+pub type ResetFinalizationParameter = Vec<AccountAddress>;
+
+#[receive(
+    contract = "election",
+    name = "resetFinalizationPhase",
+    parameter = "ResetFinalizationParameter",
+    error = "Error",
+    mutable
+)]
+fn reset_finalization_phase(ctx: &ReceiveContext, host: &mut Host<State>) -> Result<(), Error> {
+    let now = ctx.metadata().block_time();
+
+    ensure!(
+        ctx.sender().matches_account(host.state.admin_account.get()),
+        Error::Unauthorized
+    );
+    ensure!(
+        now > host.state.election_end && now > host.state.decryption_deadline,
```

Comment:
```suggestion
        now > host.state.decryption_deadline,
```
It's already ensured I think in setup that decryption deadline is later than election end.